docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,17 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+[IMPORTANT]
+====
+Using a database to store transaction logs has several benefits, but in comparison with using the file system to store the logs, you might notice a reduction in performance.
+Storing transaction logs into a database is slower, creates bigger overhead, and requires higher maintenance.
+Use this approach when prioritizing consistency over throughput.
```

Comment:
I don't think this sentence is true. Consistency must always be guaranteed in XA transactions. Could you please remove this sentence from the PR? Thanks

Reply:
+1 saying that it is slow, adds overhead and is higher maintenance depends upon how the environment is configured so it is not a true statement. If you want to provide a cost benefit analysis of the two stores you must proved a thorough balanced view of the two approaches and indicate the circumstances under which one type shines over the other. Can you also indicate under which circumstances consistency is compromised, a user would need to know, otherwise he will in all likelihood read this and opt for a transaction manager that does provide JTA semantics.